cookbooks/aws-parallelcluster-environment/test/controls/ebs_spec.rb

```diff
@@ -17,6 +17,18 @@
     its('type') { should eq 'ext4' }
     its('options') { should include '_netdev' }
   end
+
+  describe 'Verify EBS is correctly mounted'
+
+  describe bash("df -h -t ext4 | tail -n +2 | awk '{{print $2, $6}}' | grep '/shared_dir'") do
+    its('exit_status') { should eq(0) }
+    its('stdout')      { should match('976M /shared_dir') }
+  end
+
+  describe bash("cat /etc/fstab") do
+    its('exit_status') { should eq(0) }
+    its('stdout')      { should match('UUID=.* /shared_dir ext4 _netdev 0 0') }
```

Comment:
Do we still need an integration test? Could we split this control and mark its part which does not depend on how the unit test is set up with `tag:config` so that it is checked daily as part of kitchen tests?

Reply:
We still need integration tests because we're testing the different flavours of raid (0 and 1) and with multiple devices (5). In future we can extend kitchen tests and remove

Reply:
Added